contracts/FundMe.sol

```diff
@@ -4,41 +4,61 @@ pragma solidity ^0.8.0;
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 import "./PriceConverter.sol";
 
+error FundMe__NotOwner();
+
+/**
+ * @title A contract for crowd funding
+ * @author Patrick Collins
+ * @notice This contract is to demo a sample funding contract
+ * @dev This implements price feed as our library
+ */
 contract FundMe {
   using PriceConverter for uint256;
 
-  mapping(address => uint256) public s_addressToAmountFunded;
```

Comment:
Address to amount mapping shouldn't be public, added getter at the end of the contract.